src/czechpmdevs/buildertools/editors/object/FillSession.php

```diff
@@ -7,7 +7,7 @@
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *	 http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Although tabs are used for indenting, this would mess up refactoring of all the files together.
```suggestion
 *     http://www.apache.org/licenses/LICENSE-2.0
```

Reply:
Yep, my bad, I surely replaced every spaces in the file